example/src/constants/stories.ts

```diff
@@ -13,7 +13,9 @@ const stories = [
         duration: 3,
         isReadMore: true,
         storyId: 1,
-        isSeen: false
+        isSeen: false,
+        showOverlay: true,
```

Comment:
@neel-simform We should update story data and story data format to include these keys, It is also better to add a snippet and screenshot of that custom overlay in readme